.github/workflows/check.yaml

```diff
@@ -12,38 +12,39 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
-env:
-  RPOLARS_ALL_FEATURES: "true"
-
 jobs:
   R-CMD-check:
     runs-on: ${{ matrix.config.os }}
-    name: ${{ matrix.config.os }} (${{ matrix.config.r }})
+    name: ${{ matrix.config.os }} (${{ matrix.config.r }}) ${{ matrix.config.all-features && 'all-features' || '' }}
     strategy:
       fail-fast: false
       matrix:
         config:
-          #- { os: macos-latest, r: "devel" }
           - { os: macos-latest, r: "release" }
-          #- { os: macos-latest, r: "oldrel-1" }
           - { os: windows-latest, r: "devel", rtools-version: "43" }
           - { os: windows-latest, r: "release", rtools-version: "" }
           - { os: windows-latest, r: "oldrel-1", rtools-version: "42" }
           - { os: ubuntu-latest, r: "devel", http-user-agent: "release" }
           - { os: ubuntu-latest, r: "release" }
           - { os: ubuntu-latest, r: "oldrel-1" }
+        include:
+          - config: { os: macos-latest, r: "release", all-features: true }
+          - config: { os: windows-latest, r: "release", all-features: true }
+          - config: { os: ubuntu-latest, r: "release", all-features: true }
```

Comment:
@sorhawell I would like to remove R-oldrel on Windows and R-devel on Windows because the matrix is getting huge.
What do you think?

Reply:
can we keep, Windows R-devel for early warning? [I turned it of previously and regretted that decision  :)](https://github.com/pola-rs/r-polars/blob/5fe587b2492ceb54bb14355040c1a33de106b0b1/src/Makevars#L25)

I don't mind turning off oldrel

Reply:
Makes sense. I have removed R-oldrel on Windows.